Aspire BudgetingTests/Aspire_BudgetingTests.swift

```diff
@@ -6,8 +6,8 @@
 //  Copyright © 2019 TeraMo Labs. All rights reserved.
 //
 
-import XCTest
 @testable import Aspire_Budgeting
+import XCTest
 
 class Aspire_BudgetingTests: XCTestCase {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mohitathwani/aspirebudgeting/blob/63f4950fa21bc408c69d7150a9ad99db88cf17ba/Aspire BudgetingTests/Aspire_BudgetingTests.swift#L12"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>